lib/google_sign_in/helper.rb

```diff
@@ -7,54 +7,68 @@ def google_sign_in(url:, &block)
         google_sign_in_javacript_include_tag +
         google_sign_in_client_id_meta_tag
 
-      google_sign_in_javascript_tag + 
+      google_sign_in_javascript_tag +
       google_sign_in_hidden_form_tag(url: url) +
       google_sign_in_click_handler(&block)
     end
 
     private
+      HIDDEN_INPUT_ID = "google_sign_in_token"
+      SUBMIT_BUTTON_ID = "google_sign_in_submit"
+      CONTAINER_ID = "google_sign_in_container"
```

Comment:
Not sure that these constantized IDs carry their own weight. They're not really any more explanatory, shorter, or likely to be values of independent change. I think I'd vote to just inline them.